sdk/monitor/monitor-query/src/logsQueryClient.ts

```diff
@@ -89,7 +91,8 @@ export class LogsQueryClient {
           workspaceId,
           {
             query,
-            timespan
+            timespan,
+            workspaces: additionalWorkspaces?.map((x) => x)
```

Comment:
I don't think you need this `map` here right?

Reply:
will it just copy it completely?